src/Cryptol/REPL/Command.hs

```diff
@@ -589,6 +594,20 @@ refEvalCmd str = do
   val <- liftModuleCmd (rethrowEvalError . R.evaluate expr)
   rPrint $ R.ppValue val
 
+nameToNumber :: M.Name -> Int
+nameToNumber n = M.nameUnique n
```

Comment:
This seems like an odd place for this declaration, let's move it wherever `M.Name` lives.